src/bindings/python/tests/test_runtime/test_properties.py

```diff
@@ -30,6 +30,11 @@ def test_property_ro():
 
 def test_allow_auto_batching_property():
     core = Core()
+    device_name = core.get_property("CPU", "FULL_DEVICE_NAME")
+
+    if "Intel" not in device_name:
+        pytest.skip("Properties are Intel(R) CPU specific")
```

Comment:
I think it should be either:
```suggestion
@pytest.mark.skipif(os.environ.get("TEST_DEVICE", "CPU") != "CPU",
                    reason=f"Cannot run test on device {os.environ.get('TEST_DEVICE')}, Plugin specific test")
def test_allow_auto_batching_property():
    core = Core()
    device_name = core.get_property("CPU", "FULL_DEVICE_NAME")

    if "Intel" not in device_name:
        pytest.skip("Properties are Intel(R) CPU specific")
```
Or:
```suggestion
def test_allow_auto_batching_property(device):
    core = Core()
    device_name = core.get_property(device, "FULL_DEVICE_NAME")

    if device != "CPU" and "Intel" not in device_name:
        pytest.skip("Properties are Intel(R) CPU specific")
```

Reply:
fixed